src/js/view/preview.js

```diff
@@ -1,13 +1,53 @@
 define([
-    'utils/underscore'
-], function(_) {
+    'utils/underscore',
+    'utils/helpers'
+], function(_, utils) {
 
     var Preview = function(_model) {
         this.model = _model;
 
-        this.model.on('change:playlistItem', this.loadImage, this);
+        this.model.on('change:playlistItem', onPlaylistItem, this);
+        this.model.on('change:mediaModel', onMediaModel, this);
+        this.model.on('change:state', onChangeState, this);
     };
 
+    function onChangeState(model, state) {
+        var endOfPlaylist = (this.model.get('item') === (this.model.get('playlist').length - 1));
+        if ((state === 'complete') && (!model.get('repeat')) && endOfPlaylist) {
+            this.loadImage(model, model.get('playlistItem').image);
+        }
+    }
+
+    function onMediaModel() {
+        this.model.mediaModel.off('change:mediaType', onMediaType);
+        this.model.mediaModel.on('change:mediaType', onMediaType, this);
+    }
+
+    function onMediaType(mediaModel) {
+        var audio = (mediaModel.get('mediaType') === 'audio');
+        var imageLoaded = (this.model.get('item') === 0) && (!this.model.get('autostart') || utils.isMobile());
+        if (audio) {
+            // only load the image if not already loaded
+            if (!imageLoaded) {
```

Comment:
@robwalch I added this check because you mentioned that image will be loaded twice for first playlist audio item - once from onPlaylistItem, once in here. Do we want to remove this check?